env/environment-3.7.yml

```diff
@@ -40,7 +40,7 @@ dependencies:
     - ddtrace==0.50.3
     - flask_wtf==0.14.3
     - lmfit==1.0.1
-    - platon==5.3
+    - platon==4.0
```

Comment:
Just noting here that `atmospheric_retrievals` breaks with `platon>=5.0`, so I am reverting back to `4.0` here.  We could perhaps update `atmospheric_retrievals` in a separate PR.